README.md

```diff
@@ -202,6 +202,29 @@ CashRegister.drawer_position_opened # => CashRegister.where(drawer_position: 0)
 CashRegister.drawer_position_closed # => CashRegister.where(drawer_position: 1)
 ```
 
+### Note about default scopes
+
+FlexibleEnum does not play well with default scopes that reference the enum's column. For example, if you have this class:
+
+```ruby
+class User < ActiveRecord::Base
+  flexible_enum :status do
+    active 1
+    inactive 2
+  end
+
+  default_scope -> { where(status: ACTIVE) }
+end
+```
+
+`User.inactive` will result in this SQL:
```

Comment:
"attempts to use the `User.inactive` scope that FlexibleEnum provides will result in this SQL:"